Ag pure s3

agoude opened the pull request:
<!--
Hello 👋
Thank you for submitting a Pull Request.

For Work In Progress Pull Requests, please use the Draft PR feature, see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details. As long as your Pull Request is a draft, the OVHcloud Guides Team will not start proofreading it.

For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments.

DO NOT USE a fork if you are an OVHcloud employee. Instead, please contact the OVHcloud Guides Team so to be granted /write access to this repository.

Feel free to apply labels on your Pull Request, selecting them from the list to your right.

Before submitting a Pull Request, please ensure you have:

- 📖 Read the OVHcloud Documentation readme: https://github.com/ovh/docs/blob/develop/readme.md
- 🇬🇧 Edited (at least) the en-gb version of a guide.
- 🇫🇷 Edited also the fr-fr version if you are a French speaker.
- 🌐 Edited all the files if you're doing a fix or a minor update (this is not mandatory but recommended).
- 👷‍♀️ Created a small PR, if applicable.
- ✅ Tested every step you're documenting.
- 📝 Used descriptive commit messages.
- 📐 Resized images which width exceeds 1400px.
- 📝 Edited or blurred any sensitive/private information from your text and images (IPs, user IDs, private URLs, etc.).
-->

## What type of Pull Request is this?

<!-- Delete the lines which don't apply: -->
- New guide(s) 
## Description

Adding a new Pure Storage Flashblade with OVHcloud Object Storage guide

## Mandatory information

<!-- Delete the line which doesn't apply: -->
- This Pull Request can be merged as soon as possible.

<!-- If you know about the following, please mention it. If you don't know, delete the line.-->
- This Pull Request content should be replicated for the US OVHcloud documentation : YES<!-- https://support.us.ovhcloud.com/hc/en-us -->

agoude committed:
Create Flasblade EN guide.en-gb.md

New guide for Pure Storage FlashBlade With OVHcloud Object Storage

agoude committed:
Update Flashblade EN guide.en-gb.md

Creation of the content for connexion between Flashblade and OVHcloud Object Storage.
EN Guide.

agoude committed:
Create test.txt

agoude committed:
Add files via upload

Adding images

agoude committed:
Delete pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/images/test.txt

agoude committed:
Create meta.yaml

agoude committed:
Update index.md

Adding Pure Storage Flashblade and Object Storage entry

agoude committed:
Update EN guide.en-gb.md

Adding storage platforms table and Pure Storage Flashblade

agoude committed:
Update FR guide.fr-fr.md

Adding Pure Storage Flashblade

Jessica41 committed:
1st proofread guide.en-gb.md

Jessica41 committed:
Update guide.en-gb.md

Jessica41 committed:
create FR file

Jessica41 committed:
Update guide.fr-fr.md

Jessica41 commented:
Hello @agoude , I have translated the guide to FR but I am curious to know if the translation is correct for the different buttons in the GUI? or should we just keep the EN names? Thanks.

Y0Coss commented on pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.en-gb.md:
```diff
@@ -0,0 +1,81 @@
+
+### Add Remote Cloud Object Storage Credentials
+
+5. Go to the `Protection`{.action} section and then click on `Object Replica Links`{.action}.
```
It is very likely that these numbered lists separated by H3 won't work when pushed to prod.
Unless they are absolutely necessary, I'd recommend getting rid of the numbers.
The alternative would be using `1\.` `2\.` `3\.` etc.

agoude replied:
> It is very likely that these numbered lists separated by H3 won't work when pushed to prod. Unless they are absolutely necessary, I'd recommend getting rid of the numbers. The alternative would be using `1\.` `2\.` `3\.` etc.

Hi @Y0Coss I'm fine with removing them, do you want me to commit a new version?

agoude commented:
> Hello @agoude , I have translated the guide to FR but I am curious to know if the translation is correct for the different buttons in the GUI? or should we just keep the EN names? Thanks.

Hello! We should keep EN names please ;) thx

agoude committed:
Update EN guide.en-gb.md

Adding \. before each step number

agoude committed:
Update FR guide.fr-fr.md

Changelog:
- Adding \. before step number
- Keeping englisg name for reference to the GUI
- moving back to conteneur rather than "compartiments"

agoude commented:
> > Hello @agoude , I have translated the guide to FR but I am curious to know if the translation is correct for the different buttons in the GUI? or should we just keep the EN names? Thanks.
> 
> Hello! We should keep EN names please ;) thx

@Jessica41 FYI, I did commit an update, thx!

Y0Coss commented on pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md:
```diff
@@ -0,0 +1,81 @@
+
+Consultez notre guide [Object Storage - Premiers pas avec Object Storage](/pages/storage_and_backup/object_storage/s3_getting_started_with_object_storage).
+
+## Instructions
```
```suggestion
## En pratique
```

Y0Coss commented on pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md:
```diff
@@ -0,0 +1,81 @@
+
+4\. Nommez la connexion, par exemple « OVHCLOUD_OBJECT_STORAGE » et ajoutez les points de terminaison OVHcloud Object Storage associés dans le champ `Address`.
+
+![page de connexion](images/01.Connections.png){.thumbnail}
```
```suggestion
![page de connexion](images/01-connections.png){.thumbnail}
```
Images should be renamed accordingly

Y0Coss commented on pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md:
```diff
@@ -0,0 +1,81 @@
+
+![creds page](images/02.Creds.png){.thumbnail}
+
+### Créez un bucket qui sera répliqué sur l’Object Storage OVHcloud
```
```suggestion
### Créer un bucket qui sera répliqué sur l’Object Storage OVHcloud
```

Y0Coss commented on pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md:
```diff
@@ -0,0 +1,81 @@
+
+18\. Ajoutez le nom de votre conteneur Object Storage OVHcloud, « OVHcloud-dest-2025 ».
+
+19\. Ajoutez vos identifiants à distance « OVHcloud_credentials ». Le lien de réplica de conteneur va être créé.
```
To rephrase

Y0Coss commented on pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md:
```diff
@@ -0,0 +1,81 @@
+
+## Instructions
+
+### Créer une connexion cible compatible S3
```
@agoude , do we need to add the usual aws disclaimer here?

agoude replied:
this is done in my last commit

agoude committed:
Update pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

agoude committed:
Update pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

agoude committed:
Update FR guide.fr-fr.md

Changelog:
- modifying sentences to better match source meaning

agoude committed:
Update EN guide.en-gb.md

Adding S3 trademark details

agoude committed:
Update FR guide.fr-fr.md

Adding S3 trademark details

agoude committed:
Update pages/storage_and_backup/object_storage/s3_pure_storage_flashbalde/guide.fr-fr.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

agoude committed:
Update EN guide.en-gb.md

changelog:
- new images pathnames

agoude committed:
Update FR guide.fr-fr.md

changelog:
- new images pathnames

agoude commented:
@Y0Coss just need to reupload the new images with the right names and should be fine now.

agoude committed:
Adding new files

new files with correct names

agoude commented:
> @Y0Coss just need to reupload the new images with the right names and should be fine now.

now uploaded! thx

Y0Coss committed:
Final proofreading

Y0Coss commented:
Thank you for your contribution @agoude
